### DISCUSSION TITLE
Unable to run the program onto Lethal Company - A bit of help?

### DISCUSSION THREAD

**Discussion by ShadowSaiyan** [winstxnhdw/lc-hax#573] [Q&A]
So I've been using this mostly just as a form of mods for a while. Today, I went to use it and it didn't work no more. So I redownloaded all the files then ran the Requirements.Bat and waited for that to finish. Then when ran the one file in lethal it gave many errors. And now even after retrying multiple times it wasn't working still. I have 8.0 NET and Git Updated. One error was with the Bin file, it was completely empty when it should've contained something. Anyone got anything that should help me fix this? Or is this an issue with a recent update on the files?

**Comment by kenfuu** [comment]
at least I'm not the only one experiencing this, hopefully it's something that we can just fix

**Comment by winstxnhdw** [comment]
Might be my fault. Made an update to the dependency last night. I’ll check it out.

**Comment by winstxnhdw** [comment]
Should be fixed now. A bit frustrating that this error wasn't picked up by CI for some reason.

**Reply by winstxnhdw** [reply]
Okay, I figured this out. The syntax I used was valid but it's still in `preview`. I'll add this link for my own reference: https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/field-keyword

**Reply by winstxnhdw** [reply]
To use this syntax, I'll have to update this project to use .NET 9.0
